bin/controller.py

```diff
@@ -52,7 +52,10 @@ def post_new():
     except ValueError as exc:
         raise bt.HTTPError(400, str(exc))
 
-    snippet = models.Snippet.create(code, max(maxusage, 0), lifetime)
+    try:
+        snippet = models.Snippet.create(code, max(maxusage, 0), lifetime)
+    except RuntimeError as exc:
```

Comment:
is the try/catch necessary ? couldn't we just let the error bubble up ?

Reply:
Directly throw an error 500 ?

Reply:
IMO we don't need to reraise the execption, just let the RuntimeError bubbleup

![2021-01-16_00-01-23](https://user-images.githubusercontent.com/8208953/104786748-1d17be80-578e-11eb-88e8-3e8d81872eb4.png)

Reply:
I am of the same opinion.
I didn't know Bottle could do that.